src/main/java/com/vmware/vipclient/i18n/I18nFactory.java

```diff
@@ -90,7 +90,7 @@ public Message getMessageInstance(Class c, VIPCfg cfg) {
         if (null == cfg) {
             key = c.getCanonicalName();
         } else {
-            key = cfg.getProductName();
+            key = c.getCanonicalName() + "." + cfg.getProductName();
```

Comment:
Because two different instances(TranslationMessage and LocaleMessage) support shared component, if not add class name, the key for two different instances is the same, one instance maybe covered by the other.